server/sonar-server/src/test/resources/org/sonar/server/db/migrations/v52/FeedProjectLinksComponentUuidTest/schema.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE "PROJECT_LINKS" (
```

Comment:
I would argue that you could remove all the non significant columns, but it's up to you